app/views/catalog/_facets.html.erb

```diff
@@ -1,14 +1,15 @@
 <% # main container for facets/limits menu -%>
 <% if has_facet_values? %>
 <div id="facets" class="facets sidenav facets-toggleable-sm">
-  <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#facet-panel-collapse" aria-controls="facet-panel-collapse" aria-expanded="false" aria-label="Toggle facets">
-    <span class="navbar-toggler-icon"></span>
-  </button>
 
-  <div class="top-panel-heading">
-    <h2 class="facets-heading">
+  <div class="navbar">
+    <h2 class="facets-heading navbar-brand">
```

Comment:
This doesn't make sense to me.  The facets title is not a brand. Why are we giving it this css class?

Reply:
Sorry, working too quickly perhaps. Works fine without the .navbar-brand class there. I'll resubmit.